api/src/certification/session/infrastructure/storage/cpf-exports-storage.js

```diff
@@ -0,0 +1,45 @@
+import { S3ObjectStorageProvider } from '../../../../shared/storage/infrastructure/providers/S3ObjectStorageProvider.js';
+import { config } from '../../../../shared/config.js';
+import { CpfExport } from '../../domain/models/CpfExport.js';
+import { logger } from '../../../../shared/infrastructure/utils/logger.js';
+import dayjs from 'dayjs';
+import bluebird from 'bluebird';
+
+class CpfExportsStorage {
+  #client;
+
+  constructor() {
+    this.#client = S3ObjectStorageProvider.createClient(config.cpf.storage.cpfExports.client);
+  }
+
+  async sendFile({ filename, readableStream }) {
+    return this.#client.startUpload({ filename, readableStream });
+  }
```

Comment:
```suggestion
  sendFile = this.#client.startUpload;
```
y'a p'tet un monde ou on fait juste ça 🤔

Reply:
dans tous les cas ✌️ 
```suggestion
  sendFile({ filename, readableStream }) {
    return this.#client.startUpload({ filename, readableStream });
  }
```

Reply:
Je laisse le async (même retour que https://github.com/1024pix/pix/pull/7439/commits/4ae04247fc910553c1bfd7fb42f21fb45ea40e78#r1395765995 )